docker/benchmarks/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# The ideal base image would be "microsoft/dotnet-nightly:2.1.0-preview1-runtime-deps-stretch".
+# However, the wrk package is currently broken in "debian:stretch" (4.0.2-1+b1), but fixed in
+# "debian:buster" (4.0.2-2).  For now, it's easier to use "debian:buster" as the base image and 
+# manually install the .NET Core dependencies, rather than using "microsoft/dotnet-nightly" and
+# compiling wrk from source.
+
+FROM debian:buster
+
+# .NET Core dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        ca-certificates \
+        libc6 \
+        libcurl3 \
+        libgcc1 \
+        libgssapi-krb5-2 \
+        libicu57 \
+        liblttng-ust0 \
+        libssl1.0.2 \
+        libstdc++6 \
+        libunwind8 \
+        libuuid1 \
+        zlib1g \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+# KoreBuild dependencies
+        unzip \
+        wget \
+# BenchmarksClient dependencies
+        wrk \
+# BenchmarksServer dependencies
+        git \
+        procps \
+    && rm -rf /var/lib/apt/lists/*
+
+# Runtimes prior to 2.0.0 are not supported on debian:stretch (or later), but they should
+# not be required for Benchmarks.
+ENV KOREBUILD_SKIP_OLD_RUNTIME_INSTALL 1
+
+WORKDIR /benchmarks
+COPY . .
```

Comment:
I believe `src\*\*.csproj` may work (there's no recursive wildcard, it uses Go's [filepath.Match rules](https://golang.org/pkg/path/filepath/#Match)). Also, if the `sln` is present and the `csproj` is added to the `sln`, restore would fail if you added a `csproj` to the `sln` but not to the Dockerfile. So it would be a clearer failure.

Anyway, it's just optimizing build times, so it's not essential.